end-to-end-tests/direct_execute.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+export RUST_BACKTRACE=1
+alias wapm=target/debug/wapm
+wapm config set registry.url "https://registry.wapm.dev"
+
+echo "hello" | wapm execute base64
+wapm execute echo "hello"
```

Comment:
If we can do the following I think it will be cleaner

```suggestion
wax echo "hello"
```